packages/pilot/src/containers/AddRecipient/BankAccountStep/AddAccount/index.js

```diff
@@ -0,0 +1,181 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import Form from 'react-vanilla-form'
+
+import {
+  Button,
+  CardActions,
+  CardContent,
+  Col,
+  FormInput,
+  Spacing,
+  Row,
+} from 'former-kit'
+
+import accountTypes from '../../../../models/accountTypes'
+import AddAccountContent from './AddAccountContent'
+
+import createNumberValidation from '../../../../validation/number'
+import createRequiredValidation from '../../../../validation/required'
+import createMaxLengthValidation from '../../../../validation/maxLength'
+import createAccountDigitValidation from '../../../../validation/accountCheckDigit'
+import createAgencyDigitValidation from '../../../../validation/agencyCheckDigit'
+
+import style from '../style.css'
+
+const renderDocumentNumber = (data, t) => {
+  if (data.identification.documentType === 'cpf') {
+    return (
+      <Row>
+        <Col tv={2} desk={4} tablet={5} palm={8}>
+          <FormInput
+            disabled
+            className={style.marginBottom}
+            label={t('pages.add_recipient.document_owner')}
+            type="text"
+            value={data.identification.cpf}
+          />
+        </Col>
+      </Row>
+    )
+  }
+
+  return (
+    <Row>
+      <Col tv={2} desk={4} tablet={5} palm={8}>
+        <FormInput
+          disabled
+          className={style.marginBottom}
+          label={t('pages.add_recipient.document_owner')}
+          type="text"
+          value={data.identification.cnpj}
+        />
+      </Col>
+    </Row>
+  )
+}
+
+const AddAccount = ({
+  data,
+  errors,
+  onBack,
+  onCancel,
+  onContinue,
+  sharedData,
+  t,
+}) => {
+  const max30Message = t('pages.add_recipient.field_max', { number: 30 })
+  const max13Message = t('pages.add_recipient.field_max', { number: 13 })
+  const max5Message = t('pages.add_recipient.field_max', { number: 5 })
+  const numberMessage = t('pages.add_recipient.field_number')
+  const requiredMessage = t('pages.add_recipient.field_required')
+  const digitMessage = t('pages.add_recipient.field_invalid_digit')
+
+  const isNumber = createNumberValidation(numberMessage)
+  const max30Characters = createMaxLengthValidation(30, max30Message)
+  const max13Characters = createMaxLengthValidation(13, max13Message)
+  const max5Characters = createMaxLengthValidation(5, max5Message)
+  const required = createRequiredValidation(requiredMessage)
+  const isAccountDigit = createAccountDigitValidation(digitMessage)
+  const isAgencyDigit = createAgencyDigitValidation(digitMessage)
+
+  return (
+    <Form
+      data={data}
+      validateOn="blur"
+      validation={{
+        agency: [required, isNumber, max5Characters],
+        agency_digit: [isAgencyDigit],
+        bank: [required],
+        name: [required, max30Characters],
+        number: [required, isNumber, max13Characters],
+        number_digit: [required, isAccountDigit],
+        type: [required],
+      }}
+      onSubmit={(formData, formErrors) => {
```

Comment:
Poderia ser utilizado um operador unário aqui
```suggestion
      onSubmit={(formData, formErrors) => !formErrors && onContinue(formData)}
```